### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -177,29 +177,29 @@ examples-hosted: \
...
-.build/gmf-apps.timestamp: $(GMF_APPS_ALL_SRC_FILES) $(WEBPACK_CONFIG_FILES) .build/node_modules.timestamp .build/examples-hosted-gmf-apps-deps.timestamp
+.build/gmf-apps.timestamp: $(GMF_APPS_ALL_SRC_FILES) $(WEBPACK_CONFIG_FILES) .build/node_modules.timestamp
 	npm run build-gmf-apps
 	touch $@
 
-.PHONY: python-buildtools-deps
-python-buildtools-deps: .build/requests.timestamp .build/urllib3.timestamp .build/beautifulsoup4.timestamp
+.build/python-buildtools-deps.timestamp: .build/requests.timestamp .build/urllib3.timestamp .build/beautifulsoup4.timestamp
 	# needed for the generate-examples-index.py, gh-pages-cleanup and deploy helper scripts
+	touch $@
```
Comment (sbrunner) [inline Makefile]: Why this change?

File: Makefile [hunk 2]
```diff
@@ -196,157 +214,50 @@ examples-hosted-apps: \
...
+		$(addprefix contribs/gmf/build/gmf-, $(addsuffix .json, $(LANGUAGES))) \
+		$(addprefix contribs/gmf/build/angular-locale_, $(addsuffix .js, $(LANGUAGES))) \
+		$(addprefix contribs/gmf/fonts/fontawesome-webfont., eot ttf woff woff2) \
+		$(addprefix contribs/gmf/fonts/gmf-icons., eot ttf woff) \
+		$(addprefix contribs/gmf/cursors/,grab.cur grabbing.cur)
+	mkdir -p .build/examples-hosted/contribs/gmf
+	# We need the files for each app
+	# To simplify processing, we first copy them in gmfappsdeps directory, then from there to each app
+	$(foreach f,$^,mkdir -p .build/examples-hosted/gmfappsdeps/`dirname $(f)`; cp $(f) .build/examples-hosted/gmfappsdeps/$(f);)
+	cd .build/examples-hosted/gmfappsdeps/contribs/gmf; $(foreach app, $(GMF_APPS), rsync -r . ../../../contribs/gmf/apps/$(app)/;)
```
Comment (sbrunner) [inline Makefile]: Why 'gmfappsdeps' and not 'contribs/gmf/apps'?

File: Makefile [hunk 3]
```diff
@@ -455,14 +346,14 @@ contribs/gmf/apps/.tx/config: contribs/gmf/apps/.tx/config.mako $(PY_VENV_BIN)/m
 	node buildtools/extract-messages $(NGEO_PARTIALS_FILES) $(NGEO_JS_FILES) > $@
 
 .build/locale/gmf.pot: lingua.cfg .build/node_modules.timestamp \
-		$(GMF_PARTIALS_FILES) $(GMF_JS_FILES)
+		$(GMF_PARTIALS_FILES) $(GMF_SRC_JS_FILES)
```
[the remark this answers is not in the document]
Comment (gberaudo) [reply Makefile]: right, I will revert to the old name.

File: Makefile [hunk 4]
```diff
@@ -229,15 +241,15 @@ gh-pages:
 		buildtools/examples-index.mako.html \
 		$(NGEO_EXAMPLES_HTML_FILES) \
 		$(PY_VENV_BIN)/mako-render \
-		.build/beautifulsoup4.timestamp
+		python-buildtools-deps
```
[the remark this answers is not in the document]
Comment (gberaudo) [reply Makefile]: It is to simplify pip dependency management. As discussed, I will generalize to depend on a single timestamp.

**Comment by gberaudo** [comment]
@sbrunner, I addressed your comments. Could you please have a ne look?
